### ISSUE TITLE
Installation of nassl via pip on TravisCI Ubuntu Trusty boxes

### ISSUE THREAD

**Issue by jambonrose** [nabla-c0d3/nassl#18]
Hi,
I've just opened #17 about the current data quota stopping users from cloning and pulling this repo.

A big part of why we clone this repo is because we need to install it (and [sslyze](https://github.com/nabla-c0d3/sslyze)) on TravisCI. `pip install` fails on those machines (last I tried, which admittedly was on the beta Trusty machines, not on the new Trusty machines [TravisCI recently announced](https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming)).

If we enabled installation of nassl via pip on Travis' Ubuntu Trusty, our bandwidth usage of git-lfs would drop to near zero, and I suspect might help others as well.

Would you be open to a pull-request with updated binaries for Ubuntu Trusty (for testing on TravisCI)? What other distributions are these binaries aimed at?

Of course, to do this I would need to be able to interact with the repo, which requires some resolution to happen in #17.

**Comment by nabla-c0d3** [comment]
What's wrong with the current Linux binaries? To be compatible with as many Linux distros as possible, it's better to have binaries built using an older Linux.

**Comment by jambonrose** [comment]
Absolutely nothing!

My apologies for opening this issue - I was misremembering our reasons for forking nassl. When we'd tried to install in March, we'd needed unreleased features, and had been installing from the repo directly, which had resulted in gcc compilation errors. We therefore forked the project, re-compiled the binaries, and were using that fork for testing.

I've just switched the code from our branch to the 0.16.3 release on PyPI, and everything works great.

Thanks again for sslyze and nassl.

Closing this given that everything works just fine.
